ReactiveCocoa/Swift/Property.swift

```diff
@@ -107,12 +109,13 @@ public final class MutableProperty<Value>: MutablePropertyType {
 	public let producer: SignalProducer<Value, NoError>
 
 	/// Initializes the property with the given value to start.
-	public init(_ initialValue: Value) {
+	public init(_ initialValue: Value, willSet: ((value: Value) -> Value)? = { $0 }) {
 		lock.name = "org.reactivecocoa.ReactiveCocoa.MutableProperty"
 
 		(producer, observer) = SignalProducer<Value, NoError>.buffer(1)
 
-		_value = initialValue
+		self.willSet = willSet
+		_value = self.willSet(initialValue)
 		sendNext(observer, initialValue)
```

Comment:
Oops, this should send the transformed `intialValue`, right?
Which is good that we found because that means that's not covered by a test :D